bucket/materialize.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "1.78",
+    "description": "Image-to-material converter for use in video games",
+    "homepage": "http://www.boundingboxsoftware.com/materialize/",
+    "license": "GPL-3.0-or-later",
```

> GNU General Public License v3. Use it for whatever you want.

There is nothing about future versions

```suggestion
    "license": "GPL-3.0-only",
```

---

I assumed that the "or later" statement was implicit. My bad